.github/workflows/build.yml

```diff
@@ -1,6 +1,6 @@
 name: Python package
 
-on: [push, pull_request]
+on: [pull_request]
```

Comment:
Is this a change we should do in `master` as well? Or did you turn it off temporarily as you are working on a branch directly in the Equinor repo?

Reply:
Turned it of temporarily yes 🙂

Reply:
I think we maybe should consider removing it in master as well

Reply:
Thanks! I was wondering because I'm introducing another pipeline in #1452. But, I'll just keep it as the others also running on `push` and then we can make a unified decision for all of them in this PR or later :)